src/views/primary-auth/CustomButtons.js

```diff
@@ -110,7 +110,9 @@ export default View.extend({
         'data-se': options.dataAttr,
       },
       className: options.className + ' default-custom-button',
-      title: options.title,
+      title: function() {
```

Comment:
can you also check if it works as expected with idp_discovery flow

Reply:
I'm new to this widget -- do you mean a test in the same `PrimaryAuth_spec` file that has `settings` that includes both `customButtons` and also `idps` config?

Reply:
@pradeepdewda-okta I've added the same tests in the `IDPDiscovery_spec` test file